x-pack/test/upgrade/apps/dashboard/dashboard_smoke_tests.ts

```diff
@@ -24,7 +25,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
     ];
 
     const dashboardTests = [
-      { name: 'flights', numPanels: 16 },
+      { name: 'flights', numPanels: 15 },
```

Comment:
Wouldn't the number of panels in flights also be dependant on the version? The new Controls likely don't count as a `panel`

Reply:
Thanks @ThomThomson I am currently running some tests to see, will touch base when I get the results.

Reply:
So the results all passed with the new numbers, since this change is only for 8.6 - if the numbers do change per version it would be accounted for in that branch.